Examples/Tests/photon_photon_collisions/inputs_two_photons_3d

```diff
@@ -0,0 +1,137 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 100
+amr.n_cell = 16 16 16
+amr.max_grid_size = 16
+amr.blocking_factor = 4
+amr.max_level = 0
+geometry.dims = 3
+geometry.prob_lo = -1e-6 -1e-6 -1e-6
+geometry.prob_hi = 1e-6 1e-6 1e-6
+
+#################################
+##### BOUNDARY CONDITIONS #######
+#################################
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 1
+warpx.cfl = 1.0
+algo.particle_shape = 1
+
+#################################
+############ PLASMA #############
+#################################
+particles.species_names = photonA photonB electron positron
+particles.photon_species = photonA photonB
+
+photonA.species_type = photon
+photonA.injection_style = "SingleParticle"
+photonA.single_particle_pos = 0 0 0
+photonA.single_particle_u = 2.8 0 0 # gamma times beta
+photonA.single_particle_weight = 1e14
+photonA.do_not_push = 1
+photonA.do_not_deposit = 1
```

Comment:
Do we really need `do_not_deposit` ? Photons should not generate currents...

Reply:
We don't. I liked it because it's consistent with the other species. But I can remove it.

Reply:
I would be inclined to suggest removing it: these inputfiles are also used as examples, and I think that this specific line con confuse a user